app/main.py

```diff
@@ -1,3 +1,61 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+from app.shop import Shop
+from app.car import Car
+from app.customer import Customer
+
+
+def shop_trip() -> None:
+    with open("app/config.json", "r") as config_file:
+        config_data = json.load(config_file)
+        fuel_price = config_data["FUEL_PRICE"]
+        costumers_list = config_data["customers"]
+        shops_list = config_data["shops"]
+
+    for costumer in costumers_list:
+        client = Customer(
+            costumer["name"],
+            costumer["product_cart"],
+            costumer["location"],
+            costumer["money"],
+            costumer["car"],
+        )
```

Comment:
I recommend you create classmethod in Customer class, for example, load_from_dict(cls, customer).
Also fix the typo: costomer -> customer